nixos/modules/programs/steam.nix

```
@@ -139,6 +146,11 @@ in {
       Load the extest library into Steam, to translate X11 input events to
       uinput events (e.g. for using Steam Input on Wayland)
     '';
+
+    protontricks = {
+      enable = lib.mkEnableOption "Protontricks";
```

Please consider users who may not know what protontricks is. The option to enable it should offer at least a little information on it.

We could also pull it from the package's meta like this: 

```suggestion
      enable = lib.mkEnableOption "protontricks, ${cfg.protontricks.package.meta.description}";
```